fdbrpc/AsyncFileChaos.actor.h

```diff
@@ -26,94 +26,103 @@
 #include "flow/ActorCollection.h"
 #include "flow/actorcompiler.h"
 
-
-//template <class AsyncFileType>
+// template <class AsyncFileType>
 class AsyncFileChaos final : public IAsyncFile, public ReferenceCounted<AsyncFileChaos> {
 private:
 	Reference<IAsyncFile> file;
+	Arena arena;
+
 public:
 	explicit AsyncFileChaos(Reference<IAsyncFile> file) : file(file) {}
 
 	void addref() override { ReferenceCounted<AsyncFileChaos>::addref(); }
 	void delref() override { ReferenceCounted<AsyncFileChaos>::delref(); }
 
-	uint8_t toggleNthBit(uint8_t b, uint8_t n) {
-		auto singleBitMask = uint8_t(1) << (n);
-		return b ^ singleBitMask;
-	}
+	static double getDelay() {
```

Comment:
I think it would be better to resolve these global pointers at construction time and store them as members of the instance.  Then this function becomes a member function and simpler.  It's also less overhead, though that probably isn't a huge concern in a throttled state.